web/generate-geometries.js

```diff
@@ -947,6 +949,6 @@ topoMoreDetails = topojson.simplify(topoMoreDetails, 0.001);
 // Merge topoMoreDetails into topo
 mergeTopoJsonSingleZone(topo, topoMoreDetails);
 
-topo = topojson.filter(topo, topojson.filterWeight(topo, 0.009));
+//topo = topojson.filter(topo, topojson.filterWeight(topo, 0.009));
```

Comment:
The island is very small (19km2).

With the line commented, I get the island to render in the browser, with `"AUS-WA-RI":{"type":"MultiPolygon","arcs":[[[9507]]]}` in `/web/src/world.json` (size=641K)

With the line uncommented, the island does not appear in the browser and gets referred to as `"AUS-WA-RI":{"type":null}` in `/web/src/world.json` (size=547K)

I guess this code serves some purpose to reduce the size of the map to be made available to the browser, but I don't know what to do with it so this island can be rendered. You'll note that `moreDetails: true` doesn't help.

Reply:
addressed by @madsnedergaard in #2987 :)